install/install.sh

```diff
@@ -0,0 +1,287 @@
+#!/bin/bash
+#
+#
+set -o errexit -o nounset
+BINARY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+SCRIPT_DIR=$BINARY_DIR/scripts
+
+# Make sure only root can run our script
+if [[ $EUID -ne 0 ]]; then
+   echo "This script must be run as root" 1>&2
+   exit 1
+fi
+
+show_help() {
+  cat << EOF
+Installs a 
+
+USAGE: ${0##*/} [-v] CONFIG_FILE
+
+-v               verbose
+
+CONFIG_FILE     A config file for this installation.
+                See $(pwd)/template.conf
+EOF
+}
+
+VERBOSE=false
+CONFIG_FILE=""
+
+if [ "$#" -ne 1 ] || [ "$#" -eq 2 ] && [ "$1" != "-v" ] ; then
+  echo "ERROR: Illegal number of parameters!"
+  echo "Parameters: $*"
+  show_help
+  exit 1
+elif [ "$#" -eq 2 ] && [ "$1" == "-v" ] ; then
+	VERBOSE=true
+	CONFIG_FILE="$2"
+else
+	CONFIG_FILE="$1"
+fi
+
+# Check if config exists and is readable
+if  [[ ! -f "$CONFIG_FILE" ]] || [[ ! -r "$CONFIG_FILE" ]] ; then
```

Comment:
-f tests for existence of the given file. Without you are testing the string value.